src/v3/src/components/Widget/index.tsx

```diff
@@ -514,6 +514,7 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
         nonce={cspNonce}
         stylisPlugins={stylisPlugins}
       >
+        <CssBaseline />
```

Comment:
Does `OdysseyProvider` already provide a `CssBaseline`? What is the impact of and need for this change?